examples/platform-specific/cc26xx/cc26xx-web-demo/README.md

```diff
@@ -54,33 +54,35 @@ tab, as per the image below.
 
 CoAP Server
 -----------
-For this functionality to work, you will need to install the
-[Copper (Cu)](https://addons.mozilla.org/en-US/firefox/addon/copper-270430/)
-addon to your browser.
+For this functionality to work, you will need to install a CoAP client. On
+Mac OS, a guide on how to achieve this can be found
```

Comment:
maybe point to the platform-independent https://github.com/contiki-ng/contiki-ng/wiki#setting-up-contiki-ng instead ? (I added coap-cli and Mosquitto to the linux native install btw)

Reply:
Ya on it